src/core/isAttached.js

```diff
@@ -0,0 +1,25 @@
+define( [
+	"../core",
+	"../var/getShadowRoot",
+	"../var/document",
+	"../selector" // Get jQuery.contains
+], function( jQuery, getShadowRoot, document ) {
+	"use strict";
+
+	// Replace the use of contains function to check attachment (gh-3504)
+
+	var isAttached;
+
+	// Check if browser supports Shadow DOM
```

Comment:
Let's drop the preceding comment and update this to something like "Check attachment across shadow DOM boundaries when possible (gh-3504)".